dnf-behave-tests/container-test-wrapper

```diff
@@ -21,7 +21,7 @@ elif [ "$RESERVE" == "-R" ]; then
 fi
 
 TEST_EXIT=0
-behave $DEFINES "$@" || TEST_EXIT=$?
+( ./run-dbus;  behave $DEFINES "$@" ) || TEST_EXIT=$?
```

Comment:
I think we should run the dbus and polkit processes conditionally (only if we're actually running daemon tests).

It would be nice to have some control over the processes, e.g. check if they failed to start, but I suspect with them forking to the background we can't do anything. Or use some process manager, I just very quickly found this: http://garywiz.github.io/chaperone/index.html

Thinking about it, maybe we could "manage" the processes ourselves from inside behave? That is, run them and make sure they're fine for the duration of the tests.